.travis.yml

```diff
@@ -90,11 +94,14 @@ script:
     - build_lib "$PLAT" "$INTERFACE64"
 
 after_success:
-    # Upload libraries to Rackspace container
-    - pip install wheelhouse-uploader
-    # Now disabled because we have lost free
-    # services with Rackspace
-    #- python -m wheelhouse_uploader upload --local-folder
-    #     ${TRAVIS_BUILD_DIR}/libs/
-    #     --no-update-index
-    #     wheels
+    # Upload libraries to the shared staging area on anaconda.org
+    # Note that the anaconda-client package on PyPI is too old. Install from github
+    # tag instead:
+    - pip install -q git+https://github.com/Anaconda-Platform/anaconda-client@1.7.2
+    - anaconda -t $OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN upload
+            --no-progress --force -u multibuild-wheels-staging
+            -t file -p "openblas-libs"
+            -v "$(cd OpenBLAS && git describe --tags)"
+            -d "OpenBLAS for multibuild wheels"
+            -s "OpenBLAS for multibuild wheels"
+            libs/openblas*.tar.gz
```

Comment:
Yeah, might be useful to skip if secret variable is missing; I think that happens in some way with the previous setup as well; if you want new binaries uploaded the feature branch has to be in the repo proper rather than a PR originating from someone's fork.

That's probably good since we may not necessarily always want to change the hash on every PR that gets opened.